### PR TITLE
Converted contact type and candidate type to code values

### PR DESCRIPTION
**Pull request by mjtravers**
@toddlees Ready for CR.

When I tried to log into the api docker instance to do a pip uninstall fecfile-validate I hit permission issues so could not install from local. Maybe have the docker image build try and call local?

Also, did we decide on JSON file indentation of 4 spaces? I converted the 4 contact schemas to 4-spaces but noticed all the other schemas look to be 2 spaces now

### CODE REVIEW

File: schema/Contact_Candidate.json [hunk 1]
```diff
@@ -1,454 +1,394 @@
...
+            "pattern": "^[ A-Za-z0-9]{0,2}$",
+            "fec_spec": {
+                "COL_SEQ": 17,
+                "FIELD_DESCRIPTION": "CANDIDATE DISTRICT",
+                "TYPE": "A/N-2",
+                "REQUIRED": "X (error)",
+                "AUTO_POPULATE": null,
+                "SAMPLE_DATA": null,
+                "VALUE_REFERENCE": null,
+                "RULE_REFERENCE": "Required if Office = H",
```
Comment (toddlees) [inline schema/Contact_Candidate.json:354]: do these rules need to be followed this sprint?

**Review by toddlees — COMMENTED** [review]
can you add the change to the datadictionary htnl?

**Comment by mjtravers** [comment]
> can you add the change to the datadictionary htnl?

Oh, forgot. Thx for the reminder

File: schema/Contact_Candidate.json (hunk 1 above)
Comment (mjtravers) [reply schema/Contact_Candidate.json:354]: No, this rule reference is being covered by ticket web-app#119

Comment (toddlees) [reply schema/Contact_Candidate.json:354]: great!

**Review by toddlees — APPROVED** [review]

**Pull request merged**